src/inputs.ts

```diff
@@ -69,19 +64,34 @@ export function getInputs(): Inputs {
     format: core.getBooleanInput("format"),
     breaking: core.getBooleanInput("breaking"),
     breaking_against: core.getInput("breaking_against"),
-    breaking_against_config: core.getInput("breaking_against_config"),
-    breaking_limit_to_input_files: core.getBooleanInput(
-      "breaking_limit_to_input_files",
-    ),
     push: core.getBooleanInput("push"),
-    push_create: core.getBooleanInput("push_create"),
-    push_create_visibility: core.getInput("push_create_visibility"),
-    push_labels: core.getMultilineInput("push_labels"),
-    push_git_metadata: core.getBooleanInput("push_git_metadata"),
-    push_source_control_url: core.getInput("push_source_control_url"),
+    push_disable_create: core.getBooleanInput("push_disable_create"),
     archive: core.getBooleanInput("archive"),
-    archive_labels: core.getMultilineInput("archive_labels"),
+    archive_labels: [],
   };
+  if (github.context.eventName === "push") {
+    const event = github.context.payload as PushEvent;
+    core.info(`The head commit is: ${event.before}`);
+    if (inputs.breaking_against === "") {
+      inputs.breaking_against = `${event.repository.clone_url}#format=git,commit=${event.before}`;
+    }
+    console.log("BREAKING AGAINST", inputs.breaking_against);
+    inputs.archive_labels.push(github.context.ref);
+  }
+  if (github.context.eventName === "pull_request") {
+    const event = github.context.payload as PullRequestEvent;
+    core.info(`The head commit is: ${event.pull_request.head.sha}`);
+    if (inputs.breaking_against === "") {
+      inputs.breaking_against = `${event.repository.clone_url}#format=git,commit=${event.pull_request.base.sha}`;
+    }
+    console.log("BREAKING AGAINST", inputs.breaking_against);
```

Comment:
not sure if these logs were intentionally left it, but it seems like if we want to log stuff like this it would be better to just log the exact buf commands we are running when we run them

Reply:
Removed